src/clients/meta/MetaClient.cpp

```diff
@@ -37,6 +39,13 @@ DEFINE_int32(meta_client_retry_interval_secs, 1, "meta client sleep interval bet
 DEFINE_int32(meta_client_timeout_ms, 60 * 1000, "meta client timeout");
 DEFINE_string(cluster_id_path, "cluster.id", "file path saved clusterId");
 DEFINE_int32(check_plan_killed_frequency, 8, "check plan killed every 1<<n times");
+DEFINE_uint32(failed_login_attempts,
+              5,
+              "how many consecutive incorrect passwords cause the account to become locked");
+DEFINE_uint32(password_lock_time,
```

Comment:
password_lock_time_in_secs

Reply:
MySQL uses the day as the unit of password_lock_time, and this is because MySQL supports unblocking a user manually. Since we don't have this ability right now, I use hours. But seems using seconds is sufficient for the block time window and will make testing easier. I'll change it in the next commit.